cmd/buildah/config.go

```diff
@@ -107,6 +107,32 @@ var (
 	}
 )
 
+func updateEntrypoint(builder *buildah.Builder, c *cli.Context) {
+	if len(strings.TrimSpace(c.String("entrypoint"))) == 0 {
+		builder.SetEntrypoint(nil)
+		return
+	}
+	var entrypointJSON []string
+	err := json.Unmarshal([]byte(c.String("entrypoint")), &entrypointJSON)
+
+	if err == nil {
+		builder.SetEntrypoint(entrypointJSON)
```

Comment:
I don't know that is expected behaviour, as long as it is documented in the man page.  We could drop this down to logrus.Warningf then it will not give the user noice.

Reply:
It appears to be the current behaviour of Docker, as shown here:
https://github.com/projectatomic/buildah/pull/741#discussion_r192418132

In this example, 'pixdrift' is being passed to echo if echo is declared in exec form (array).

From Docker documentation (although it's a little ambiguous):
https://docs.docker.com/engine/reference/builder/#entrypoint

> You can use the exec form of ENTRYPOINT to set fairly stable default commands and arguments and then use either form of CMD to set additional defaults that are more likely to be changed.
```
FROM ubuntu
ENTRYPOINT ["top", "-b"]
CMD ["-c"]
```

Reply:
My punctuation was bad on my previous reply.  I know this is the correct and expected behaviour.  I Will drop it down a level so user is not flooded with the message.

Reply:
np, suggestion sounds good 🍺